apps/recovery-utility/renderer/components/Modals/WithdrawModal/InitiateTransaction/index.tsx

```diff
@@ -128,13 +144,19 @@ export const InitiateTransaction = ({ accountsArray, assetsInAccount, initialAcc
         <TextField
           id='toAddress'
           label='Recipient Address'
-          error={errors.to?.message}
           autoComplete='off'
           autoCapitalize='off'
           spellCheck={false}
           isMonospace
+          error={!!errors.to}
+          inputProps={{
+            sx: {
```

Comment:
I'm just being picky but Do we have a style component we're using? Maybe something instead of 'Red'? 

Also we could create a custom hook for this for better readability but we don't have to. 

export const useTextFieldStyles = (theme: Theme, hasError: boolean): SxProps => ({
  ...(hasError && {
    borderColor: theme.palette.error.main,
  }),
  fontFamily: 'Monospace', // if you want to apply monospace to all TextFields
});


  const textFieldStyles = useTextFieldStyles(theme, !!errors.to);
  
<TextField>
   inputProps={{ sx: textFieldStyles }}
</TextField>

Reply:
@dainkang1 we currently don't but we do have examples of using themes.
We might want to take the approach you suggested when we move towards QOL changes 
@ArielSir-fb  please see [this](https://github.com/fireblocks/recovery/blob/destination-address-validation/apps/recovery-utility/renderer/components/RecoveryForm/index.tsx#L195) as an example and adjust.

Reply:
@dainkang1 this should now be resolved. Address validation moved to be part of TransactionInitInput check in [this commit](https://github.com/fireblocks/recovery/pull/35/commits/5f5f3a9791d80fd5b01a73f001b932fbdcc82f74).